[gRPC][Stub Generator] Error while downloading protoc executor in windows

**Description:**
Error occurred while downloading protoc executor in windows. Following error thrown,

[2018-07-09 15:30:05,252] ERROR {org.ballerinalang.launcher.Main} - Error while generating protoc executable.  
org.ballerinalang.protobuf.exception.BalGenToolException: Error while generating protoc executable. 
	at org.ballerinalang.protobuf.cmd.GrpcCmd.execute(GrpcCmd.java:130)
	at java.util.Optional.ifPresent(Unknown Source)
	at org.ballerinalang.launcher.Main.main(Main.java:66)
Caused by: org.ballerinalang.protobuf.exception.BalGenToolException: Error saving file 'C:\Users\wso2\Desktop\ballerina-examples\examples\proto-to-ballerina\protoc-windows-x86_64.exe'.
	at org.ballerinalang.protobuf.utils.BalFileGenerationUtils.saveFile(BalFileGenerationUtils.java:213)
	at org.ballerinalang.protobuf.cmd.GrpcCmd.downloadProtocexe(GrpcCmd.java:261)
	at org.ballerinalang.protobuf.cmd.GrpcCmd.execute(GrpcCmd.java:127)
	... 2 more
Caused by: java.io.FileNotFoundException: http://repo1.maven.org/maven2/com/google/protobuf/protoc/3.4.0\protoc-3.4.0-windows-x86_64.exe
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
	at java.net.URL.openStream(Unknown Source)
	at org.ballerinalang.protobuf.utils.BalFileGenerationUtils.saveFile(BalFileGenerationUtils.java:204)
	... 4 more

Problem in protoc download url.

**Steps to reproduce:**
Run proto-ballerina example in windows machine.

**Affected Versions:**
0.975.0

**OS, DB, other environment details and versions:**
Windows

--- a/misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/cmd/GrpcCmd.java
+++ b/misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/cmd/GrpcCmd.java
@@ -23,7 +23,6 @@ import com.beust.jcommander.Parameters;
 import org.ballerinalang.launcher.BLauncherCmd;
 import org.ballerinalang.net.grpc.builder.BallerinaFileBuilder;
 import org.ballerinalang.net.grpc.exception.BalGenerationException;
-import org.ballerinalang.protobuf.BalGenerationConstants;
 import org.ballerinalang.protobuf.exception.BalGenToolException;
 import org.ballerinalang.protobuf.utils.BalFileGenerationUtils;
 import org.slf4j.Logger;
@@ -254,9 +253,8 @@ public class GrpcCmd implements BLauncherCmd {
                 } catch (IOException e) {
                     throw new BalGenToolException("Exception occurred while creating new file for protoc exe. ", e);
                 }
-                String url = PROTOC_PLUGIN_EXE_URL_SUFFIX + protocVersion + BalGenerationConstants.FILE_SEPARATOR +
-                        "protoc-" + protocVersion + "-" + OSDetector
-                        .getDetectedClassifier() + PROTOC_PLUGIN_EXE_PREFIX;
+                String url = PROTOC_PLUGIN_EXE_URL_SUFFIX + protocVersion + "/protoc-" + protocVersion + "-" +
+                        OSDetector.getDetectedClassifier() + PROTOC_PLUGIN_EXE_PREFIX;
                 try {
                     saveFile(new URL(url), exePath);
                     File file = new File(exePath);

